### PR TITLE
PCGenStatusBar: reduce width of comically oversized button

### PR DESCRIPTION
**Pull request by grimreaper**

### CODE REVIEW

File: code/src/java/pcgen/gui2/PCGenStatusBar.java [hunk 1]
```diff
@@ -128,13 +131,17 @@ else if (nwarnings > 0)
...
 				Tooltip tooltip = new Tooltip(String.format(
 						"%d errors and %d warnings occurred while loading the sources",
 						finalNerrors,
 						finalNwarnings
 				));
 				loadStatusButton.setTooltip(tooltip);
+				loadStatusButton.setGraphicTextGap(0.0);
+				loadStatusButton.setContentDisplay(ContentDisplay.GRAPHIC_ONLY);
+				loadStatusButton.setAlignment(Pos.CENTER_RIGHT);
+//				loadStatusButton.set
```
Comment (karianna) [inline code/src/java/pcgen/gui2/PCGenStatusBar.java]: commented out code

File: code/src/java/pcgen/gui2/PCGenStatusBar.java [hunk 2]
```diff
@@ -62,19 +66,18 @@
...
 		add(messageLabel);
 		add(Box.createHorizontalGlue());
 		progressBar.setStringPainted(true);
 		progressBar.setVisible(false);
 		add(progressBar);
 		add(Box.createHorizontalGlue());
-		add(GuiUtility.wrapParentAsJFXPanel(loadStatusButton));
+		JFXPanel wrappedButton = GuiUtility.wrapParentAsJFXPanel(loadStatusButton);
+		//todo: calculate this rather than hard code
+		wrappedButton.setMaximumSize(new Dimension(750, 20000000));
```
Comment (karianna) [inline code/src/java/pcgen/gui2/PCGenStatusBar.java]: I think we should calculate this before landing this PR

**Comment by LegacyKing** [comment]
@grimreaper ETA to complete this?

**Comment by LegacyKing** [comment]
Can we get this completed @grimreaper ?

**Comment by LegacyKing** [comment]
@karianna I don't think this is going to be resolved.

File: code/src/java/pcgen/gui2/PCGenStatusBar.java (hunk 2 above)
Comment (LegacyKing) [reply code/src/java/pcgen/gui2/PCGenStatusBar.java]: @karianna I'm not sure this is going to get handled. Are we going to merge?

**Pull request merged**